src/UmbracoFileSystemProviders.Azure/AzureBlobFileSystemConfig.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace Our.Umbraco.FileSystemProviders.Azure
+{
+    public class AzureBlobFileSystemConfig
+    {
+        public bool DisableVirtualPathProvider { get; set; }
+
+        /// <summary>
+        /// The container name
+        /// </summary>
+        public string ContainerName { get; set; }
+
+        /// <summary>
+        /// The root url
+        /// </summary>
+        public string RootUrl { get; set; }
+
+        /// <summary>
+        /// The connection string
+        /// </summary>
+        public string ConnectionString { get; set; }
+
+        //TODO: INT - The current underlying AzureBlobFileSystem expects this as string - YUK
+        /// <summary>
+        /// The maximum number of days to cache blob items for in the browser
+        /// </summary>
+        public string MaxDays { get; set; }
```

Comment:
This is because the Pseudo-DI that V7 uses when parsing the configuration originally passes the parameters as strings. Change them to the real deal anytime.